README.md

```diff
@@ -45,32 +45,63 @@ All this operations can be more easily accomplished using the whisper-client lib
 
 # Try it yourself
 
-From the project root folder, fire the following commands to execute this project in development mode:
+From the project root folder, fire the following commands to execute this project in development mode.
+
+1. Up the applications that whisper need
+
+    ```bash
+    docker-compose up -d local
+    ```
+
+2. Compile the local version
+
+    ```bash
+    go build
+    ```
+
+3. Serve whisper locally
+
+    This will serve the auxiliary services (databases and Hydra) and will run Whisper at the `7070` port; endpoints `/login` and `/consent` will display our incredibly simple user interface.
+
+    The only way to access this endpoints is through a valid authorization url.
+
+    **With some small Dockerfile trickery, it is possible to override the provided UI files to use your custom page layout and icons.**
+
+    ```bash
+    ./whisper serve --port 7070 \
+        --base-ui-path ./web/ui/www \
+        --hydra-admin-url http://localhost:4445 \
+        --hydra-public-url http://localhost:4444 \
+        --secret-key uhSunsodnsuBsdjsbds \
+        --log-level debug \
+        --scopes-file-path ./scopes.json \
+        --database-url "mysql://root:secret@tcp(localhost:3306)/whisper?charset=utf8mb4&parseTime=True&loc=Local"
+    ```
+ 
+4. Create the endpoint with hydra for testing
+    
+    They only work once! It will be necessary to run again.
+    
+    ```bash
+    docker-compose exec hydra \                                    
+        hydra clients create \  
```

Comment:
Please add the following comment: This command will launch the registered client application above at `http://localhost:5555`. Go to your browser and see if you can successfully login with it.